aerogear/graphback
# Templates fix

Fixing couple very small bugs in templates;


- [x] Added ability to run and play with the templates without linking graphback versions. If we really would need non production versions then we can disable package.json by renaming it to something else.
I think this might be controversial change but it is really painful to work with templates now without it.

- [x] Hidden implementation details of the codegen and graphback commands. 
- [x] Moved all code to use _id to reuse the same app
- [x] [GraphQL-Migrations]: Fixes an issue where "Primary key with `ID` ScalarType and field name different to `id` isn't auto-incrementable"

## Verification
0. yarn install
1. Run all templates together with the client app.
2. Create Notes and comments. 

After that we can push that to 0.16.0 and verify it in production 🤣
